src/uid2Sdk.ts

```diff
@@ -347,6 +421,21 @@ export class UID2 {
           console.warn(`UID2 callbacks on identity event failed.`, reason)
       );
   }
+
+  private async callCstgAndSetIdentity(
+    request: { emailHash: string } | { phoneHash: string },
+    opts: ClientSideIdentityOptions
+  ) {
+    const cstgResult = await this._apiClient!.callCstgApi(request, opts);
```

Comment:
what happens if this call is rejected?

Reply:
An exception gets thrown to the user of the SDK. My thinking was to leave it up to them to deal with retries (or not) however they see fit. We can always add in some retry functionality later, but I didn't want to add it until someone asks for it.